site/en/guide/embedding.md

```diff
@@ -211,11 +211,14 @@ vectors in the embedding variable, except for the header.  Consequently, the
 variable.  If the TSV metadata file has only a single column, then we don’t
 expect a header row, and assume each row is the label of the embedding. We
 include this exception because it matches the commonly-used "vocab file"
-format.
+format.Also, when you open the load data tab in the embedding visualizer there
+it is specified about what is TSV file of vector and what is the TSV file
+of metadata with an example.
```

Comment:
Clean up. Is this accurate:
The embedding visualizer's *Load Data* tab specifies the TSV file for the vector and metadata.

Reply:
i'll correct this